unity/Holo/Assets/Editor/ModelPreprocessing/AssetBundles/AssetBundleCreator.cs

```diff
@@ -48,4 +47,13 @@ private void BuildMapABs()
 
         buildMapArray = new AssetBundleBuild[1] {buildMap};
     }
+
+    private void SaveAssetBundles()
+    {
+        if (!AssetDatabase.IsValidFolder(@"Assets\StreamingAssets"))
+        {
+            AssetDatabase.CreateFolder("Assets", "StreamingAssets");
+        }
+        BuildPipeline.BuildAssetBundles(Application.dataPath + "/StreamingAssets", buildMapArray, BuildAssetBundleOptions.None, BuildTarget.StandaloneWindows);

```

Comment:
`BuildTarget` should be `WSAPlayer` most pobably. Once this is confirmed on Holo please change it.